packages/core/src/browser/window/default-window-service.ts

```diff
@@ -41,8 +41,9 @@ export class DefaultWindowService implements WindowService, FrontendApplicationC
         });
     }
 
-    openNewWindow(url: string): undefined {
-        window.open(url, undefined, 'noopener');
+    openNewWindow(url: string, options?: NewWindowOptions): undefined {
+        const features = !!options?.opener ? '' : 'noopener';
+        window.open(url, options?.windowName, features);
```

Comment:
What should happen when someone executes the code in electron env?

https://github.com/eclipse-theia/theia/blob/9c2af7ceb7a0108150996788f80b5b9f10250980/packages/core/src/electron-browser/window/electron-window-service.ts#L28-L31

Reply:
these files should be reverted